app/src/main/java/com/example/places/ListFragment.kt

```diff
@@ -20,28 +25,42 @@ class ListFragment : Fragment(R.layout.list_fragment) {
         savedInstanceState: Bundle?
     ): View? {
         binding = ListFragmentBinding.inflate(inflater, container, false)
+        binding2 = ListItemBinding.inflate(inflater,container,false)
         return binding?.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        viewModel.placesLiveData.observe(viewLifecycleOwner, { places ->
-            setupAdapter(places)
-        })
+        viewModel.placesLiveData.observe(viewLifecycleOwner) { places ->
+            setupAdapter(places) }
+
+        viewModel.placesLiveData.observe(viewLifecycleOwner) { places ->
+            for (place in places) {
+              Glide.with(this)
+                  .load(place.imageUrl)
+                  .override(600,600)
+                  .into(binding2.placeImageView)
+            }
+        }
+
     }
 
     fun setupAdapter(placesList: List<Place>) {
         val listAdapter = ListAdapter(
-            placesList
+            placesList, this
         )
-
         binding?.placesRecyclerView?.adapter = listAdapter
         binding?.placesRecyclerView?.layoutManager = LinearLayoutManager(context)
+        binding?.placesRecyclerView?.addItemDecoration(DividerItemDecoration(this.context,DividerItemDecoration.VERTICAL))
```

Comment:
You can simple add a View to your xml file that handles every item. Something like

`<View 
height = 1dp
width = match parent
background = the color you want it to be
constrainted to the bottom of the parent
/>`

That way, there will be a horizontal line that is 1 dp in height below every single item of your recyclerview.